hudi-common/src/main/java/org/apache/hudi/internal/schema/utils/SerDeHelper.java

```diff
@@ -144,6 +148,11 @@ private static void toJson(Types.RecordType record, Integer maxColumnId, Long ve
       if (field.doc() != null) {
         generator.writeStringField(DOC, field.doc());
       }
+      // NOTE: The value of null field is JsonProperties.NULL_VALUE.
+      if (field.getDefaultValue() != null && field.getDefaultValue() != JsonProperties.NULL_VALUE) {
+        generator.writeFieldName(DEFAULT_VALUE);
+        generator.writeObject(field.getDefaultValue());
```

Comment:
Do I need to verify the correctness of the defaultValue type before serialization, rather than just serializing it directly
  private static void writeDefaultValue(Types.Field field, JsonGenerator generator) throws IOException {
    if (field.getDefaultValue() == null || field.getDefaultValue() == JsonProperties.NULL_VALUE) {
      return;
    }
    switch (field.type().typeId()) {
      case RECORD:
      case ARRAY:
      case MAP:
        JsonNode defaultNode = JacksonUtils.toJsonNode(field.getDefaultValue());
        generator.writeObjectField(DEFAULT_VALUE, defaultNode);
        break;
      case STRING:
        generator.writeStringField(DEFAULT_VALUE, field.getDefaultValue().toString());
        break;
      case INT:
        generator.writeNumberField(DEFAULT_VALUE, Integer.valueOf(field.getDefaultValue().toString()));
        break;
      case LONG:
        generator.writeNumberField(DEFAULT_VALUE, Long.valueOf(field.getDefaultValue().toString()));
        break;
      case FLOAT:
        generator.writeNumberField(DEFAULT_VALUE, Double.valueOf(field.getDefaultValue().toString()));
        break;
      case DOUBLE:
        generator.writeNumberField(DEFAULT_VALUE, Double.valueOf(field.getDefaultValue().toString()));
        break;
      case BOOLEAN:
        generator.writeBooleanField(DEFAULT_VALUE, Boolean.valueOf(field.getDefaultValue().toString()));
        break;
      case DECIMAL:
        generator.writeBinaryField(DEFAULT_VALUE, (byte[])field.getDefaultValue());
        break;
      case FIXED:
        generator.writeBinaryField(DEFAULT_VALUE, (byte[])field.getDefaultValue());
        break;
      case BINARY:
        generator.writeBinaryField(DEFAULT_VALUE, (byte[])field.getDefaultValue());
        break;
      case DATE:
        generator.writeNumberField(DEFAULT_VALUE, Integer.valueOf(field.getDefaultValue().toString()));
        break;
      case TIME:
        generator.writeNumberField(DEFAULT_VALUE, Long.valueOf(field.getDefaultValue().toString()));
        break;
      case TIMESTAMP:
        generator.writeNumberField(DEFAULT_VALUE, Long.valueOf(field.getDefaultValue().toString()));
        break;
      case UUID:
        generator.writeStringField(DEFAULT_VALUE, field.getDefaultValue().toString());
        break;
      default:
        throw new RuntimeException("not implemented");
    }
  }